### ISSUE TITLE
Development status of the simulators

### ISSUE THREAD

**Issue by Wolepo** [FrSkyRC/ETHOS-Feedback-Community#3110]
Hello, if you are dependent on the simulators for tests, it is painfully obvious that many control elements do not yet work. Therefore my request is to at least make the trim buttons and function keys functional in 1.5.0. Thank you for your understanding.
Wolfgang

**Comment by hrenz** [comment]
Trims work at V1.5.0 with US Layout
but not the FS1--FS6

**Comment by Wolepo** [comment]
No, all trim buttons (T1 to T6) and none of the function switches work. The sticks are still not working properly either, they jump to 0% when going from -80 to -100% and then have to be set to "+120"% in order to be able to set -100% afterwards. A further touch in the minus direction returns to 0%.

**Comment by Wolepo** [comment]
Hello,
after much trial and error and with the help of the above drawing, I have found the corresponding key combinations for the trimmers for the X18S and with a German keyboard (QWERTZ). The trimmer T1 remains open. I have not found anything sensible for this. Can Bertrand help?
![Tastaturbelegung X18S](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/90325526/b8cbac10-2c01-4b11-846d-a07445407463)

**Comment by bsongis-frsky** [comment]
Here it is, with a full keyboard. Note that the numbers for trims are the numbers on top of letters on your keyboard
![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/50834778/4f2336c7-689d-432d-8d4f-50faa59fe3b4)
